WordOfTheDay.Repository/WordsRepository.cs

```diff
@@ -64,23 +60,21 @@ public async Task PostWord(Word word)
         }
         public Task<bool> IsAlreadyExist(Word word)
         {
-            var time = DateTime.Today.ToUniversalTime();
-
             var exist = _context.Words
-                .AnyAsync(w => w.Email == word.Email && w.AddTime > time);
+                .LaterThan()
+                .AnyAsync(w => w.Email == word.Email);
```

Comment:
Let's put this code to extension also. For me, it looks like e.g. _context.Words.LaterThan(today).ByEmail(email).Any() is more readable, because it tells what it does using a nature language.

Reply:
done